lib/modules/manager/cargo/extract.spec.ts

```diff
@@ -15,6 +15,12 @@ const cargo4toml = Fixtures.get('Cargo.4.toml');
 const cargo5toml = Fixtures.get('Cargo.5.toml');
 const cargo6configtoml = Fixtures.get('cargo.6.config.toml');
 const cargo6toml = Fixtures.get('Cargo.6.toml');
+const cargo7toml = Fixtures.get('Cargo.7.toml');
+const cargo7lock = Fixtures.get('Cargo.7.lock');
+const cargo8toml = Fixtures.get('Cargo.8.toml');
+const cargo8lock = Fixtures.get('Cargo.8.lock');
+const cargo9toml = Fixtures.get('Cargo.9.toml');
+const cargo9lock = Fixtures.get('Cargo.9.lock');
```

Comment:
do we really need so many new fixtures? can't we reuse existing or update existing fixtures?

Reply:
I have reduced the number of fixtures as much as possible and have moved them to a new folder to further reduce visual clutters.